ament_copyright/setup.py

```diff
@@ -0,0 +1,36 @@
+from setuptools import find_packages
+from setuptools import setup
+
+setup(
+    name='ament_copyright',
+    version='0.0.0',
+    packages=find_packages(exclude=['test']),
+    install_requires=['setuptools'],
+    author='Dirk Thomas',
+    author_email='dthomas@osrfoundation.org',
+    maintainer='Dirk Thomas',
+    maintainer_email='dthomas@osrfoundation.org',
+    url='https://github.com/ament/ament_lint/wiki',
```

Comment:
Are you going to create a wiki for it? Or should you just point to the repo landing page and make use of the `README`?